scripts/create_classic_core_mapping_yaml.py

```diff
@@ -0,0 +1,29 @@
+from cognite.neat import NeatSession, get_cognite_client
+from pathlib import Path
+from rich import print
+THIS_FOLDER = Path(__file__).resolve().parent
+
+XLSX_FILE = THIS_FOLDER / "core_classic_mapping.xlsx"
```

Comment:
Not sure 100% how XLSX and YAML differ

Reply:
They don't. Xlsx is better for to work in, but is slow to load so do not want to include an xlsx file in the core of the code base.